From 68c485cf985098a879894e0c10539f5982aabc9e Mon Sep 17 00:00:00 2001
From: Cai Song <cais.fnst@cn.fujitsu.com>
Date: Tue, 1 Aug 2017 05:28:37 -0400
Subject: [PATCH] net.tcp_cmds/ipneigh01: specify the specific network device

The target machine(dragonboard-410c) cannot connect to the test server
with command "ping6".
Now, specify the specific network device when using "ping6Y"

Signed-off-by: Cai Song <cais.fnst@cn.fujitsu.com>
---
 testcases/network/tcp_cmds/ipneigh/ipneigh01.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/network/tcp_cmds/ipneigh/ipneigh01.sh b/testcases/network/tcp_cmds/ipneigh/ipneigh01.sh
index 99bab16..8af81f5 100755
--- a/testcases/network/tcp_cmds/ipneigh/ipneigh01.sh
+++ b/testcases/network/tcp_cmds/ipneigh/ipneigh01.sh
@@ -41,7 +41,7 @@ do_test()
 
 	for i in $(seq 1 $NUMLOOPS); do
 
-		ping$TST_IPV6 -q -c1 $rhost > /dev/null
+		ping$TST_IPV6 -q -c1 -I${LHOST_IFACES} $rhost > /dev/null
 
 		local k
 		local ret=1
-- 
2.7.4

